app/controllers/api/v1/users/descriptions_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::Users::DescriptionsController < Api::V1::BaseController
+  
+  def create
+    describee = User.find(params[:id])
+    descriptions = JSON.parse(request.body.read)
+    describer    = User.find(descriptions['describer_id'])
+    describee.create_descriptions(descriptions['johari'], describer)
+    if describee.all_descriptions_completed?(describer.id, descriptions['johari'])
+      Assignment.complete(describee.id, describer.id)
```

Comment:
@Dpalazzari Yea that makes sense with the postman. My other thought is someone just manually entering the url as `johari/:id` and resubmitting (like if they submitted and then went back to the page and then submitted again). So I guess higher priority would be that we check there is an open assignment before letting any descriptions be added.